tutorials/dnssec-cloud-dns-domains/index.md

```diff
@@ -0,0 +1,1074 @@
+registrars” throughout.
+
+## Costs
+
+This tutorial uses Google Cloud DNS, which has a monthly cost for the existence
+of each managed zone, as well as a per-query cost. Use the [Pricing
+Calculator][6] (scroll down on that page to see an estimate for Cloud DNS with
+one zone and up to several tens of thousands of incoming queries) to generate a
+cost estimate based on your projected usage.
+
+[6]: https://cloud.google.com/products/calculator/#id=cb34e9f9-3838-48c2-9346-3d20379aa285
+
+Although validating resolvers generate additional queries (for DNSKEY records)
+on DNSSEC-activated domains, the DNSKEY queries are negligible compared to
+normal traffic. If you enable DNSSEC with NSEC denial of existence (rather than
+the default NSEC3) recursive resolvers may be able to answer queries for
+non-existent domains without additional queries to your name servers, which
+might reduce the total number of queries (especially for denial-of-service
+attacks that send millions of queries for random non-existent subdomains).
+
+## Activate DNSSEC
+
+Once you have enabled DNSSEC for your managed zone in Google Cloud DNS, you must
+activate DNSSEC through the domain registrar in order for resolvers to enforce
+DNSSEC by validating signatures in DNS responses. Do this by creating a DS
+record for your domain in the parent TLD, adding your domain to a DNSSEC chain
+of trust that between your domain and its TLD, and beyond that to the “trust
+anchor” for the root DNSKEY.
+
+### Confirm DNSSEC support
+
+#### TLD DNSSEC support
+
+Although over 90% of top-level domains support DNSSEC, some do not, and it isn’t
+possible to activate DNSSEC for a domain in a TLD that does not support it.
+Confirm whether the TLD for your domain supports DNSSEC by looking for it in the
+[ICANN TLD DNSSEC Report][7].
+
+[7]: http://stats.research.icann.org/dns/tld_report/
+
+-   If your TLD answers YES to the first two questions (Signed? and DS in Root?)
+    it supports DNSSEC (these rows are **green**).
+
+-   Most TLDs that do not support DNSSEC (as of October 2017) are two-letter
+    country-code domains (ccTLDs) such as .AI or .AL, but there are also a few
+    global TLDs like .AERO that do not support it (these rows are light gray).
+
+-   Some TLDs have enabled DNSSEC signing, but have not published a DS record in
+    the root zone (their rows are blue in the report). In theory you could add a
+    DS record to such a TLD and have DNSSEC activate automatically when they
+    eventually publish a DS record, but it is unlikely that any domain registrar
+    would support that, and it is not recommended, since you should be aware
+    (and in control) of any change to the DNSSEC status of your domains.
+
+If you cannot reach the ICANN TLD DNSSEC site for any reason, you can check with
+the [Verisign DNSSEC debugger][8]: just enter the TLD name and hit return; if
+you see any red ⊗ errors the TLD does not support DNSSEC (e.g.
+<http://dnssec-debugger.verisignlabs.com/aq>).
```

Comment:
This would be stripped out, it looks like HTML

Reply:
Fixed.